<ui version="4.0" >
 <class>CustomScheme</class>
 <widget class="QWidget" name="CustomScheme" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QListWidget" name="m_formatsListView" />
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QGroupBox" name="m_formatGroupBox" >
       <property name="title" >
        <string>&amp;Format</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="m_boldCheckBox" >
          <property name="text" >
           <string>&amp;Bold</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QCheckBox" name="m_overLineCheckBox" >
          <property name="text" >
           <string>&amp;Overline</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QCheckBox" name="m_strikeOutCheckBox" >
          <property name="text" >
           <string>&amp;Strikeout</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="m_italicCheckBox" >
          <property name="text" >
           <string>&amp;Italic</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="m_underLineCheckBox" >
          <property name="text" >
           <string>&amp;Underline</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QCheckBox" name="m_waveUnderLineCheckBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;WaveUnderline</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_colorGroupBox" >
       <property name="title" >
        <string>&amp;Color</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Foreground</string>
          </property>
          <property name="buddy" >
           <cstring>m_foreGroundComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="XinxColorCombo" name="m_foreGroundComboBox" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Background</string>
          </property>
          <property name="buddy" >
           <cstring>m_backGroundComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="XinxColorCombo" name="m_backGroundComboBox" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_exampleGroupBox" >
       <property name="title" >
        <string>&amp;Example</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XinxColorCombo</class>
   <extends>QComboBox</extends>
   <header>xinxcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_underLineCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_waveUnderLineCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>330</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
